src/makeCalculator.js

```diff
@@ -3,8 +3,35 @@
 /**
  * @return {object}
  */
+
+
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+    add(a, b) {
+      return a + b;
```

Comment:
[CODE KNOWLEDGE] - Since these methods are meant to be used as callbacks for the `operate` method, they should only take one argument and perform the operation on the `result` property. For instance, the `add` method should look like `add(number) { this.result += number; }`.